test/test_google.py

```diff
@@ -73,14 +73,6 @@ def get_csv_file_contents(file_id):
 
 ###############################################
 # Tests
-def test_open_access_dashboard_folder_id():
-    folder_id = google.open_access_dashboard_folder_id()
-    assert folder_id is not None
-
-
-def orcid_dashboard_folder_id():
-    folder_id = google.orcid_dashboard_folder_id()
-    assert folder_id is not None
```

Comment:
these methods don't exist anymore in the google class, each publish step defines the relevant name directly now